proxywasm/internal/hostcall_utils_tinygo.go

```diff
@@ -26,6 +26,9 @@ import (
 )
 
 func RawBytePtrToString(raw *byte, size int) string {
+	if size == 0 {
+		return ""
+	}
```

Comment:
why we need this?

Reply:
Thanks - I think I ran into a problem here but probably changed approach at some points and stopped needing